### ISSUE TITLE
afer 0.27.0 decorators supplied through Api.__init__ no longer work

### ISSUE THREAD

**Issue by tadams42** [miLibris/flask-rest-jsonapi#150]
Following code:

```python
from flask_rest_jsonapi import Api
from flask_jwt_extended import jwt_required

api = Api(decorators=(jwt_required,))
```

no longer causes JWT validation meaning I'm able to access endpoints with or without JWT.

[Documentation](https://flask-rest-jsonapi.readthedocs.io/en/latest/api.html) still claims it should work.

In 0c212bd71e9f46ab633d93703bc29c1009222554 we see that using supplied decorators is removed but it is not clear why.

- What is alternative way of decorating all endpoints with one or more decorators?
- Can you please document this or at least remove no longer true docs from `api.rst`?

**Comment by fmorato** [comment]
Indeed, that commit should be reverted unless there is a good alternative.
One alternative is to register the `Api()` as a blueprint and apply a decorator to [`before_request`](http://flask.pocoo.org/docs/1.0/api/?highlight=before_request#flask.Blueprint.before_request):

```python
bp = Blueprint("api", __name__)
@bp.before_request
@decorator
def before_request():
    """ Apply decorator to all endpoints. """
    pass
api = Api(app, bp)
```
https://stackoverflow.com/questions/37323089/how-to-apply-decorator-to-all-blueprint-urls-in-flask

**Comment by igieon** [comment]
Any follow ups. It will be good just to remove it form documentaiton if it not plan to support it
